.travis.yml

```diff
@@ -6,29 +6,28 @@ go:
     - 1.6
     
 env:
-    - BUILDER_VERSION=0.11.0
+    global:
+        - BUILDER_VERSION=0.11.0
+    matrix:
+        - TEST_SUITE=unit
+        - TEST_SUITE=integration
 
 cache:
     directories:
-    - ./builder-cache
+        - ./builder-cache
 
 notifications:
     email: false
 
 before_install:
-- cp ./builder-cache/builder-$BUILDER_VERSION ./builder || wget https://downloads.giantswarm.io/builder/$BUILDER_VERSION/builder && chmod +x ./builder
-- export PATH=$PATH:$PWD
-- go get -u github.com/golang/lint/golint
+    - openssl aes-256-cbc -K $encrypted_314659c9f31c_key -iv $encrypted_314659c9f31c_iv -in inago-integration-test.pem.enc -out ./inago-integration-test.pem -d
+    - cp ./builder-cache/builder-$BUILDER_VERSION ./builder || wget https://downloads.giantswarm.io/builder/$BUILDER_VERSION/builder && chmod +x ./builder
+    - chmod +x ./builder
+    - export PATH=$PATH:$PWD
+    - go get -u github.com/golang/lint/golint
 
 script:
-- make lint
-- make ci-test
-- make ci-build
-- ./inagoctl
+    - ./ci.sh
```

Comment:
I don't love it, but we need a conditional over `TEST_SUITE`. This means that we launch two jobs, due to the matrix, and the script then decides what commands to run based on `TEST_SUITE`. Can't do that in just the script section.